### PR TITLE
Formatting fixed in compliance with PEP8. Handle view,url path and form created .

### LINKED ISSUE #5: Add handle_complain view and url for staff users
Adding view for handling complains for staff users

### PR DESCRIPTION
**Pull request by Aviral14**
Close #5,#10

### CODE REVIEW

**Comment by iamkroot** [comment]
Good to see you guys have started following the guidelines well. Some improvements/nitpicks:
1. PR title should explain what is being achieved in the PR, not just closes or fixes or adds. That part should be added in the PR message body.
2. The [last](https://github.com/ccit-bphc/ccit-portal/pull/6/commits/6d5657fd3e00dc44df58aa3ab467676d1ca4c802) commit message doesn't explain anything.
3. Continuing from point 2, in this case, the overall work done in each commit is pretty less. A general trend in most orgs is to create small commits during the PR development (like you have done), and then squash them all into one (or more, depending on the commit size/impact) commit(s) when merging. Make sure to have proper commit messages in this case.

File: complaints/forms.py [hunk 1]
```diff
@@ -15,13 +15,14 @@ class Meta:
...
-        labels = {
-            "remark": ("Complaint Details"),
-            "contact_no": ("Contact Number"),
-            "room_no": ("Room Number"),
-            "avail_start_time": ("Complaint Details"),
-        }
-        widgets = {"contact_no": forms.Textarea(attrs={"cols": 20, "rows": 1})}
+        
+
+class Handle_form(forms.ModelForm):
```
Comment (iamkroot) [inline complaints/forms.py]: Class names should be `PascalCase`

File: complaints/forms.py [hunk 2]
```diff
@@ -15,13 +15,14 @@ class Meta:
...
+    class Meta:
+        model=models.Complaint
```
Comment (iamkroot) [inline complaints/forms.py]: Have spaces between operators.

File: complaints/forms.py (hunk 1 above)
Comment (Aviral14) [reply complaints/forms.py]: Don't we use Snake case in python and Pascal cases in Java?

File: complaints/forms.py (hunk 2 above)
Comment (Aviral14) [reply complaints/forms.py]: ok, will remember that!

File: complaints/forms.py (hunk 1 above)
Comment (iamkroot) [reply complaints/forms.py]: Not for class names. See [PEP8](http://www.python.org/dev/peps/pep-0008/)

**Comment by Aviral14** [comment]
#8 has some of the fixes applied.The project will be made PEP8 compliant as we work on the files.
So for now we can close this pr and merge #8

**Comment by iamkroot** [comment]
> So for now we can close this pr and merge #8

No no no no. That's not how it's done. Finish the work of this PR here only. We will work on merging this one first.

**Comment by iamkroot** [comment]
Here's how you should proceed:
1) checkout [this](https://github.com/Aviral14/ccit-portal/tree/handle) branch locally.
2) Apply your formatting fixes.
3) Squash all the commits into one using interactive rebase. (Don't forget to have proper commit message in the final commit)
4) Force push.

**Comment by Aviral14** [comment]
@iamkroot  check now!

**Comment by iamkroot** [comment]
> 1. PR title should explain what is being achieved in the PR, not just closes or fixes or adds. That part should be added in the PR message body. (Basically, exchange your title and body of the PR)

Needs to be done.

**Comment by iamkroot** [comment]
The commit message is still not proper. Think about it this way: When you're looking through the logs, the message should give a one liner about the main change that happened in the commit. Is "Fixed formatting" important enough to be in that history?

File: complaints/forms.py [hunk 3]
```diff
@@ -1,8 +1,11 @@
+"""Module containing Model forms for receiving and  handling complaints """
 from django import forms
 from . import models
 
 
-class Complaint_form(forms.ModelForm):
+class ComplaintForm(forms.ModelForm):
+    """ A Model form for receiving complaints"""
```
Comment (iamkroot) [inline complaints/forms.py]: Avoid unnecessary/self-explanatory docstrings.

Comment (Aviral14) [reply complaints/forms.py]: ok

File: complaints/forms.py [hunk 4]
```diff
@@ -15,17 +18,19 @@ class Meta:
...
-            "remark": ("Complaint Details"),
-            "contact_no": ("Contact Number"),
-            "room_no": ("Room Number"),
-            "avail_start_time": ("Complaint Details"),
-        }
-        widgets = {"contact_no": forms.Textarea(attrs={"cols": 20, "rows": 1})}
 
 
-class OpenSite_form(forms.ModelForm):
+class HandleForm(forms.ModelForm):
```
Comment (iamkroot) [inline complaints/forms.py]: `Handle` what? Class names should have specific nouns. I'd suggest something like `ComplaintForm`

File: complaints/models.py [hunk 5]
```diff
@@ -1,7 +1,8 @@
+"""Module for Creating Complaints Model"""
 from django.db import models
 from django.contrib.auth import get_user_model
 
-User = get_user_model()
+USER = get_user_model()
```
Comment (iamkroot) [inline complaints/models.py]: The earlier variable naming scheme was correct according to the Django conventions. What was wrong, is the use of globals.

Comment (Aviral14) [reply complaints/models.py]: ok .I will fix that

File: complaints/forms.py (hunk 4 above)
Comment (Aviral14) [reply complaints/forms.py]: Actually what we plan to do is that the ComplainForm will be used by the user while registering the complaints and the HandleForm will be used by the staff for changing status of the complain or for giving any remark to the user,once the complaint has been resolved.

**Comment by Aviral14** [comment]
@iamkroot Please review.

**Pull request merged**